Cesium.Runtime/StdLibFunctions.cs

```diff
@@ -15,4 +18,14 @@ public static void Exit(int exitCode)
     {
         RuntimeHelpers.Exit(exitCode);
     }
+
+    public static int Rand()
+    {
+        return shared.Next(RAND_MAX);
+    }
+
+    public static void SRand(uint seed)
+    {
+        shared = new Random(unchecked((int)seed));
```

Comment:
We compile stdlib in unchecked mode by default (and have no plans to change that), so explicit `unchecked` isn't necessary.